### PR TITLE
Docs: sections order  improvement

### PR DESCRIPTION
**Pull request by anki-code**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -2,16 +2,10 @@
...
-* python V3.6 or later.
-* for interactive use, an ansi (vt100-compatible) terminal application.  On Windows 10, the 
-  separately-installable `Windows Terminal app <https://github.com/microsoft/terminal/releases>`_ 
-  is recommended.
-  
 Installation
 ------------
-Xonsh can be installed via package manager, via appimage or just run from a pre-built container:
+
+Xonsh can be installed via different ways:
```
Comment (scopatz) [inline docs/installation.rst:8]: The original sentence here sounds better.

File: docs/installation.rst [hunk 2]
```diff
@@ -20,18 +14,11 @@ Xonsh can be installed via package manager, via appimage or just run from a pre-
     packages
     appimage
     containers
-    editors
     jupyter
 
-Configuration and Customization
-================================
```
Comment (scopatz) [inline docs/installation.rst:27]: This heading separation is useful

File: docs/installation.rst (hunk 1 above)
Comment (anki-code) [reply docs/installation.rst:8]: The text just duplicate the list:

Screenshot_20201009_022820

File: docs/installation.rst (hunk 2 above)
Comment (anki-code) [reply docs/installation.rst:27]: Here I'm not removing only header. I remove this section and move the 3 links from it to the "Config Files and Settings" sections. It's consistent.

File: docs/installation.rst (hunk 1 above)
Comment (anki-code) [reply docs/installation.rst:8]: Okay I replaced it to "Xonsh can be installed via different package managers from python specific to operation system package managers."

File: docs/installation.rst [hunk 3]
```diff
@@ -2,16 +2,10 @@
...
-* python V3.6 or later.
-* for interactive use, an ansi (vt100-compatible) terminal application.  On Windows 10, the 
-  separately-installable `Windows Terminal app <https://github.com/microsoft/terminal/releases>`_ 
-  is recommended.
-  
 Installation
 ------------
-Xonsh can be installed via package manager, via appimage or just run from a pre-built container:
+
+Xonsh can be installed via different package managers from python specific to operation system package managers:
```
Comment (scopatz) [inline docs/installation.rst:8]: probably, "Xonsh can be installed & run with various package managers, by using appimage, or from a docker container:"

File: docs/installation.rst (hunk 2 above)
Comment (scopatz) [reply docs/installation.rst:27]: Oh I see, thanks!

File: docs/installation.rst (hunk 3 above)
Comment (anki-code) [reply docs/installation.rst]: Changed. Thanks!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**